src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -502,4 +502,41 @@ class RawTableRelationTest
 
     assert(source.count() == dest.count())
   }
+
+  it should "fail reasonably when table does not exist" in {
+    val source = spark.read
+      .format("cognite.spark.v1")
+      .option("apiKey", writeApiKey)
+      .option("type", "raw")
+      .option("database", "datybasy")
+      .option("table", "assets")
+      .option("inferSchema", "false")
+      .load()
+    source.createTempView("table_which_does_not_exist")
+
+    // This query causes Spark to crash in Jetfire, let's try how does it work in here
+
+    val query =
+      """
+        WITH
+          raw_table AS (
+            SELECT
+            from_json(columns,
+            '''
+            externalId string
+            ''') as json
+            FROM table_which_does_not_exist
```

Comment:
is this the same as `cdf_raw('db_which_does_not_exist', 'table_which_does_not_exist')`

Reply:
aha sorry got it now